Api/Functions/Comments.cs

```diff
@@ -0,0 +1,94 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Attributes;
+using Microsoft.Extensions.Logging;
+using Microsoft.OpenApi.Models;
+using MovieFiles.Core.Interfaces;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+
+namespace MovieFiles.Api.Functions
+{
+    public class Comments
+    {
+        private readonly ILogger<Comments> _logger;
+        private readonly ICommentRepository _commentRepository;
+
+        public Comments(ILogger<Comments> log, ICommentRepository commentRepository)
+        {
+            _logger = log;
+            _commentRepository = commentRepository;
+        }
+        [FunctionName("GetComments")]
+        [OpenApiOperation(operationId: "GetComments", tags: new[] { "Comments" })]
+        [OpenApiParameter(name: "movieId", In = ParameterLocation.Path, Required = true, Type = typeof(int))]
+        [OpenApiParameter(name: "page", In = ParameterLocation.Path, Required = true, Type = typeof(int))]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(List<Core.Models.Comment>))]
+        [OpenApiParameter(name: "x-functions-key", In = ParameterLocation.Header, Required = true, Type = typeof(string), Description = "The function key")]
+
+        public async Task<IActionResult> GetComments(
+            [HttpTrigger(AuthorizationLevel.Function, "get", Route = "comments/{movieId}/{page}")] HttpRequest req,
```

Comment:
I think we can discuss if the page parameter will be in route this way or as query so it will be consistent

Reply:
sure